data/sheragi/the emerald sword.txt

```diff
@@ -0,0 +1,854 @@
+	
+	to offer
+		has "Sheragi Archaeology: The Emerald Sword 2: active"
+	
+	on offer
+		require "Jump Drive" 2
+		conversation
+			`You have a jump drive in your possession. Would you like to give it to Albert's crew?`
+			choice
+				`    (Yes.)`
+				`    (No.)`
+					defer
+			`	You give Albert a call, and arrange to meet with him to plan the next step. He shows up at your ship, greets you, and starts loading in some equipment along with the jump drive you brought. He has assembled a crew of around 20 people, including him and the three Hai archaeologists, now comfortable to let go of their disguise once they're back in your ship.`
+			`	"<first>, meet the people who will help us recover the Emerald Sword. I've just briefed them on the situation and what we plan to do. We're going to take some specialized tools with us, along with the jump drive, that will allow us to repair and outfit the ship in any world we can land on. I suggest you also hire some additional crew to man the ship once we capture it since it's not going to pilot itself. Remember, this is an alien warship, so I would recommend erring on the side of caution by bringing along around 100 extra crew members to deal with any potential problems in piloting the ship. If you're ready to go, here are the coordinates to Zubenelhakrabi."`
+			`	You thank Albert, and make your preparations for the voyage into the unknown.`
+				accept
+	
+	on accept
+		fail "Sheragi Archaeology: The Emerald Sword 2"
+		outfit "Jump Drive" -1
+		event "zubenelhakrabi discovered"
+	
+	on enter "Zubenelhakrabi"
+		conversation
+			`The trip to Zubenelhakrabi is completely normal and calm. Given everyone's trepidating expectations though, the trip's uneventfulness only manages to keep everyone on edge. When you arrive, a simple scan reveals the system's star is unremarkable, nearing the end of its main sequence phase. Among the planets orbiting it, one is in the habitable zone and shows clear signs of life, covered in dark green spots under the clouds.`
+			scene "scene/emeraldswordderelict"
+			`	You also find the Emerald Sword, but strangely, it is not on the planet, but drifting through space. An enormous ship of angular design, its most striking feature is an incredibly large cannon going through the center of the ship. It is so large it might as well be said the ship was built around it; you can only wonder about its power. Thankfully, you won't have to find out the hard way; the ship is inert, derelict, ruined from lack of maintenance and abandoned to the mercy of micrometeorite showers and decay. It looks like it could fall apart if looked at the wrong way... luckily, you have just the right crew for the job.`
```

Comment:
`it might as well be said the ship was built around it`
Rewording to `the ship might as well have been built around it` might sound better?
Also, need a comma after `maintenance`.